### ISSUE TITLE
different version number and no peusodobam

### ISSUE THREAD

**Issue by walterdevalk** [pachterlab/kallisto#194]
I was experimenting with kallisto and noticed that two installations act differently with your test dataset. Also the version number seems incorrect.

```bash
$> ls
reads_R1.fastq.gz  reads_R2.fastq.gz

$> kallisto version
kallisto, version 0.44.0

$> conda list | grep kallisto  # different version number
kallisto                  0.45.0               hdcc98e5_0    bioconda

$> kallisto quant \
  --index=/mnt/lab-in/references/transcriptome/interim_GRCh37.p13_rna.kallisto.idx \
  --output-dir=reads_kallisto_output \
  --plaintext \
  --fusion \
  --pseudobam \
  --genomebam \
  --threads=1 \
  --gtf=/mnt/lab-in/references/annotation/interim_GRCh37.p13_top_level_2017-01-13.gff3 \
  reads_R1.fastq.gz reads_R2.fastq.gz &> reads.kallisto.log

$> ls reads_kallisto_output  # no pseudo bam and no error message in previous command
abundance.tsv  fusion.txt  pseudoaln.bin  run_info.json
```

```bash
$> conda install kallisto=0.44.0   # downgrade

$> conda list | grep kallisto
kallisto                  0.44.0               h7d86c95_2    bioconda

$> kallisto version
kallisto, version 0.44.0

$> kallisto quant   --index=/mnt/lab-in/references/transcriptome/interim_GRCh37.p13_rna.kallisto.idx  \
  --output-dir=reads_kallisto_output2 \
  --plaintext   --fusion   --pseudobam   \
  --genomebam  \
  --threads=1   \
  --gtf=/mnt/lab-in/references/annotation/interim_GRCh37.p13_top_level_2017-01-13.gff3   
  reads_R1.fastq.gz reads_R2.fastq.gz &> reads.kallisto.log

$> reads_kallisto_output2  # now there is a bam file
abundance.tsv  fusion.txt  pseudoalignments.bam  pseudoalignments.bam.bai  run_info.json

```

**Comment by laurabogar** [comment]
Thank you for posting this! I had the same issue while running kallisto (ostensibly v. 0.44.0, actually 0.45.0) on a CentOS virtual machine. Downgrading to kallisto 0.44.0 solved the problem, as you illustrated above. I hope the --pseudobam flag works again soon in the newer versions of kallisto!

**Comment by timodonnell** [comment]
Just hit this also. Anyone know what `pseudoaln.bin` is? Can it be converted to a bam?

**Comment by CatherineJaneReynolds** [comment]
I have just had this problem - but I have found that the bam files are written fine if I include the -b bootstraps argument.

**Comment by pmelsted** [comment]
Fixed in version 0.45.1
